src/apps/bridge/sensor_drivers/borealisSensor.cpp

```diff
@@ -79,28 +234,33 @@ void BorealisSensor::borealisSubCallback(uint64_t node_id, const char *topic,
   (void)version;
   BmErr err = BmEINVAL;
   Borealis_t *borealis = NULL;
-  abstractSensorType_e sensor_type = SENSOR_TYPE_UNKNOWN;
+  BorealisSubscriptions_t sub_type = BOREALIS_SUB_COUNT;
+  static constexpr char subtag_spectrum[] = "/aos/borealis/spectrum";
+  static constexpr char subtag_levels[] = "/aos/borealis/levels";
+  static constexpr char subtag_level_statistics[] = "/aos/borealis/level_statistics";
+  static constexpr char subtag_recstatus[] = "/aos/borealis/recstatus";
```

Comment:
@matt001k These are duplicated here and around line 61. Let's dry it up by moving it to the top of the file.